src/mode/modes.ts

```diff
@@ -63,7 +64,12 @@ export class SearchInProgressMode extends Mode {
   }
 
   getStatusBarText(vimState: VimState): string {
-    return `/${vimState.globalState.searchState!.searchString}`;
+    const leadingChar =
```

Comment:
I think this would be an error if `globalState.searchState` is undefined. Can we add a check for that, if it is undefined, log an error and return an empty string and return null or something?

Reply:
sure thing. i added a check/logger call in 1c3a82f .